src/js/components/SkipLinks.js

```diff
@@ -0,0 +1,75 @@
+// (C) Copyright 2014-2015 Hewlett-Packard Development Company, L.P.
+
+var React = require('react');
+var Layer = require('./Layer');
+var Menu = require('./Menu');
+var DOM = require('../utils/DOM');
+var IntlMixin = require('../mixins/GrommetIntlMixin');
+
+var SkipLinks = React.createClass({
+
+  mixins: [IntlMixin],
+
+  componentDidMount: function () {
+    var anchorElements = document.querySelectorAll('[data-skip-label]');
```

Comment:
Actually, I'm doing some more testing now and it seems the virtual DOM may not have enough affect on the real DOM. Stay tuned.